FabricExample/__tests__/focused-input-handler.spec.tsx

```diff
@@ -17,7 +22,7 @@ function WhatUserTyped() {
     },
   });
 
-  return <Text testID='text'>{text}</Text>;
+  return <Text testID="text">{text}</Text>;
```

Comment:
Can't we force double quotes everywhere?

In this PR import-statements have single quote, jsx string props have double quotes, plain string have single quotes.

I'd love to see the consistency here and use double quotes if possible (it may have a conflicts with existing eslint/linter rules, but feel free to change it)

P. S. I love double quotes because in java single and double quotes have different meaning (symbol vs string) so I like to have a consistency between two languages

Reply:
Sure, the current prettier config is `"singleQuote": true`, let's switch to the default (`false`)